Libraries/Tile/Tile.h

```diff
@@ -0,0 +1,195 @@
+#ifndef Tile_h
+#define Tile_h
+
+class Tile{
+  private:
+    char datos1, datos2;
+
+  public:
+    Tile();
+
+    //getters
+    //datos1
+    bool getVisited();
+    bool getStart();
+    bool getCheckpoint();
+    bool getBlack();
+    bool getNorth();
+    bool getEast();
+    bool getSouth();
+    bool getWest();
+    //datos2
+    bool getRampUp();
+    bool getRampDown();
+    bool getAccessible();
+    bool getDeployedKit();
+    int getWeight();
```

Comment:
Comment each function saying what it does. i.e.
// It returns true if the current tile is visited, false otherwise.
// This value is stored on datos1.